source/release-notes/dbtools-100.12.0-changelog.txt

```diff
@@ -0,0 +1,31 @@
+.. |tool-version| replace:: 100.12.0
```

Comment:
I think we want this content directly on `database-tools-changelog.txt` and then we can delete this file. Sorry about this, I realize it's kind of a hassle. However, it we leave it like this, there's a small issue on the site where 
the **Next** navigation button points to 100.12.0. 

<img width="890" alt="Screenshot 2025-04-03 at 5 48 49 PM" src="https://github.com/user-attachments/assets/8b4dcc47-f363-4a52-8af7-128a97d8b6b1" />